tests/framework_graphql/test_application.py

```diff
@@ -147,9 +151,10 @@ def test_query_and_mutation(target_application):
         ("GraphQL/operation/%s/mutation/<anonymous>/%s" % (framework, mutation_path), 1),
     ]
     _test_query_scoped_metrics = [
-        ("GraphQL/resolve/%s/storage" % framework, 1),
         ("GraphQL/operation/%s/query/<anonymous>/storage" % framework, 1),
     ]
+    if not VERSION_3_3:
+        _test_query_scoped_metrics.append(("GraphQL/resolve/%s/storage" % framework, 1))
```

Comment:
What happened here? Is this resolver trace somehow gone?

Reply:
Whoops.  That's residual from the previous nonsense.  Fixed.